inscriptions/ISic020700.xml

```diff
@@ -0,0 +1,236 @@
+                <respStmt>
+                    <name xml:id="JCh" ref="http://orcid.org/0000-0001-6823-0265">James Chartrand</name>
+                    <resp>site construction and encoding</resp>
+                </respStmt>
+                <respStmt>
+                    <name xml:id="VV" ref="http://orcid.org/0000-0002-9695-0240">Valeria Vitale</name>
+                    <resp>editing of geo data</resp>
+                </respStmt>
+                <respStmt>
+                    <name xml:id="MM">Michael Metcalfe</name>
+                    <resp>museum data collection</resp>
+                </respStmt>
+                <respStmt>
+                    <name xml:id="SS" ref="https://orcid.org/0000-0003-3914-9569">Simona Stoyanova</name>
+                    <resp>standardisation of template and tidying up encoding</resp>
+                </respStmt>
+                <respStmt>
+                    <name xml:id="system">system</name>
+                    <resp>automated or batch processes</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>I.Sicily</authority>
+                <idno type="filename">ISic020700</idno>
+                <idno type="TM"></idno>
+                <idno type="EDR"/>
+                <idno type="EDH"/>
+                <idno type="EDCS"></idno>
+                <idno type="PHI"></idno>
+                <idno type="URI">http://sicily.classics.ox.ac.uk/inscription/ISic020700</idno>
+                <idno type="DOI"></idno>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by/4.0/">Licensed under a Creative Commons-Attribution 4.0 licence.</licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc>
+                    <msIdentifier><!--Default country and region-->
+                        <country>Italy</country>
+                        <region>Sicily</region>
+                        <settlement>Ragusa</settlement>
+                        <repository role="museum" ref="http://sicily.classics.ox.ac.uk/museum/091">Museo Archeologico Ibleo</repository>
+                        <!--No inventory number found-->
+                        <!--Adding location for old id numbers if available-->
+                        <altIdentifier>
+                            <settlement/>
+                            <repository/>
+                            <idno type="old"/>
+                        </altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <textLang mainLang="scx">Sikel<certainty locus="value" match="../@mainLang" assertedValue="grc"><desc>Ancient Greek</desc></certainty></textLang> 
+                        <!-- I start wondering if we need something different from this to encode inscriptions that are evidently the result of Greek-Sikel or Greek-Elymian linguistic 
```

Comment:
well, it comes down to whether you want to say it's Greek with Sikel interference (a Sikel speaker trying to write in Greek?), Sikel with Greek loan word, or simply to be agnostic... ultimately it's a different version of the same problem, and is a subjective judgement...?